README.md

```diff
@@ -283,6 +283,8 @@ And finally, you are ready to go! Or at least you should be...
 
 ### Junior Contributors
 
+Joan Lloret(JDLloret):| I tried so hard and got it wrong
```

Comment:
Nice try but we must keep working on this!

The name must be placed in `**` after this it will be in **bold**, then place || the nickname in the following format:`[nickname][nickname]` after this another | and place your phrase.